libs/src/services/ethContracts/delegateManagerClient.js

```diff
@@ -0,0 +1,28 @@
+const GovernedContractClient = require('../contracts/GovernedContractClient')
+const DEFAULT_GAS_AMOUNT = 200000
```

Comment:
is there a reason to redefine these? i think it's already defined with a default in ethWeb3Manager

Reply:
could use a lot of cleanup. seems like a lot of places we just hard code the gas amount of 100000. would you be ok if this was a separate PR to organize all that?

Reply:
yeah i also think gas amount should get passed as a parameter to txs instead of set within the function itself, this allows us to easily change it later on